LoopKitUI/Models/ChartAxisValueDoubleUnit.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  ChartAxisValueDoubleUnit.swift
```

Comment:
Moved because it's needed by GlucoseChart, which is needed for the insulin model chart